modules/fb_instant_articles_display/src/DrupalInstantArticleDisplay.php

```diff
@@ -2,9 +2,11 @@
 
 /**
  * @file
- * Defines the InstantArticleNodeWrapper class.
+ * Contains \Drupal\fb_instant_articles_display\DrupalInstantArticleDisplay.
  */
 
+namespace Drupal\fb_instant_articles_display;
```

Comment:
With this class now being namespaced, need to use fully qualified name for `DateTime` class throughout, ie `\DateTime` (lines 97 & 103)